lib/improver/tests/utilities/test_cube_checker.py

```diff
@@ -80,6 +80,19 @@ def test_no_x_coordinate(self):
         with self.assertRaisesRegex(ValueError, msg):
             check_for_x_and_y_axes(sliced_cube)
 
+    def test_pass_dimension_requirement(self):
+        """Test that the expected exception is raised, if there the x and y
+        coordinates are not dimensional coordinates."""
+        check_for_x_and_y_axes(self.cube, require_dim_coords=True)
```

Comment:
The docstring of this function is copied from the "fail dimension requirement" function. 
Should be more like:
"""Pass in compatible cubes that should not raise an exception. No
assert statement required as any other input will raise an exception."""

Reply:
✅